lib/socket.js

```diff
@@ -142,17 +142,26 @@ module.exports = class UDX extends events.EventEmitter {
     return binding.udx_napi_send_buffer_size(this._handle, size)
   }
 
-  send (buffer, offset, length, port, ip, onflush) {
+  send (buffer, offset, length, port, ip, ttl, onflush) {
     if (this.closing) throw new Error('Socket is closed')
     if (!this.bound) this.bind(0)
 
+    if (typeof ttl === 'function') {
+      onflush = ttl
+      ttl = undefined
+    }
+
     const id = this._allocSend()
     const req = this._reqs[id]
 
     req.buffer = buffer.subarray(offset, offset + length)
     req.onflush = onflush || noop
 
-    binding.udx_napi_send(this._handle, req.handle, id, buffer, port, ip)
+    if (ttl) {
+      binding.udx_napi_send_ttl(this._handle, req.handle, id, buffer, port, ip, ttl)
+    } else {
+      binding.udx_napi_send(this._handle, req.handle, id, buffer, port, ip)
+    }
```

Comment:
Isn't this equi? 

``` js
binding.udx_napi_send_ttl(this._handle, req.handle, id, buffer, port, ip, ttl || 0)
```

Then we can remove the normal send method from the bindings also

Reply:
A TTL of 0 will be rejected with `EINVAL` so I've made the binding switch to `udx_send()` when TTL is 0.